website/content/docs/commands/plugin/runtime/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+page_title: plugin runtime - Command
+description: |-
+  The "plugin runtime" command groups subcommands for interacting with
+  Vault's plugin runtimes, configuring the context within which external
+  plugins can be run.
+---
+
+# plugin runtime
+
+The "plugin runtime" command groups subcommands for interacting with
+Vault's plugin runtimes, configuring the context within which external
+plugins can be run.
+
+To use a registered plugin runtime, use the `-runtime` option with the
```

Comment:
It might be worth commenting about common ones being `runc` and `runsc`, what they are, and if they are available by default without registering, or if they need to be registered first?

Reply:
Yeah good idea. Added a paragraph on this below.